docs/guides/use_checkpointing.md

```diff
@@ -1,67 +1,85 @@
 ---
-jupytext:
-  formats: ipynb,md:myst
-  text_representation:
-    extension: .md
-    format_name: myst
-    format_version: 0.13
-    jupytext_version: 1.13.8
-kernelspec:
-  display_name: Python 3 (ipykernel)
-  language: python
-  name: python3
+jupyter:
+  jupytext:
+    formats: ipynb,md
+    text_representation:
+      extension: .md
+      format_name: markdown
+      format_version: '1.3'
+      jupytext_version: 1.13.8
+  kernelspec:
+    display_name: Python 3 (ipykernel)
+    language: python
+    name: python3
 ---
 
-# Checkpointing with `flax.training.checkpoints`
+# Save / Load Checkpoints
```

Comment:
Nit:

```suggestion
# Save / load checkpoints
```

or "Save and load..."

(with "sentence case" https://developers.google.com/style/capitalization#capitalization-in-titles-and-headings)